frontend/webpack.config.ts

```diff
@@ -39,6 +39,8 @@ const config: Configuration = {
     progress: true,
     hot: HOT_RELOAD !== 'false',
     inline: HOT_RELOAD !== 'false',
+    host: '0.0.0.0',
+    disableHostCheck: true,
```

Comment:
I don't think we want this by default. Can you add an env var like we use for `HOT_RELOAD` instead?

Reply:
Does it work passing these as `--host` and `--disable-host-check` flags to `yarn run dev`?